src/components/Quests/GlobalQuest.js

```diff
@@ -0,0 +1,4 @@
+export const questStatus = {
+    quest1: true,
+    quest2: false,
+  }
```

Comment:
No, that's not a good practice. We will use context to hold this state instead, similarly like the authContext is holding the `auth` prop.